stages/test/test_skopeo.py

```diff
@@ -125,3 +126,45 @@ def test_skopeo_copy(tmp_path, stage_module, dest_type, local_output_path):
 
         assert data["config"].get("digest") is not None, \
             "'manifest.json' seems corrupt - no 'config.digest' section found"
+
+@pytest.mark.parametrize("remove_signatures", [(True), (False),])
+def test_skopeo_copy_remove_signatures(tmp_path, stage_module, remove_signatures):
+    # We are completely faking everything here, because we mock out skopeo
+    fake_container_dst = tmp_path / "fake-container"
+    inputs = {
+        "images": {
+            "path": fake_container_dst,
+            "data": {
+                "archives": {
+                    fake_container_dst: {
+                        "format": "oci-archive",
+                        "name": "test-image",
+                    },
+                },
+            },
+        },
+    }
+
+    fake_skopeo = textwrap.dedent("""\
+    #!/bin/sh
+    exit 0
+    """)
+
+    with testutil.mock_command("skopeo", fake_skopeo) as args:
```

Comment:
Just passing `""` to `mock_command()` should be fine, so:
```python
diff --git a/stages/test/test_skopeo.py b/stages/test/test_skopeo.py
index a0adf13e..bcbc1c6a 100644
--- a/stages/test/test_skopeo.py
+++ b/stages/test/test_skopeo.py
@@ -145,12 +145,7 @@ def test_skopeo_copy_remove_signatures(tmp_path, stage_module, remove_signatures
         },
     }
 
-    fake_skopeo = textwrap.dedent("""\
-    #!/bin/sh
-    exit 0
-    """)
-
-    with testutil.mock_command("skopeo", fake_skopeo) as args:
+    with testutil.mock_command("skopeo", "") as args:
         options = {
             "destination": {
                 "type": "oci",
```
will also work and is slightly shorter. the default is exactly this (just doing nothing :)